resources/assets/js/stories/H2Components/Alert.stories.tsx

```diff
@@ -0,0 +1,175 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { withIntl } from "storybook-addon-intl";
+import { select, text } from "@storybook/addon-knobs";
+import { action } from "@storybook/addon-actions";
+import Alert from "../../components/H2Components/Alert";
+import { colorOptions } from "./utils";
+
+const stories = storiesOf("H2 Components/Alert", module).addDecorator(withIntl);
+
+const positionOptions: { [key: string]: "toast" | "static" } = {
+  toast: "toast",
+  static: "static",
+};
+
+stories.add(
+  "Toast Alert",
+  (): React.ReactElement => (
+    <section>
+      <Alert
+        color={select("color", colorOptions, "stop")}
+        position={select("position", positionOptions, positionOptions.toast)}
+        dismissBtn={
+          <Alert.DismissBtn onClick={action("Dismiss")}>x</Alert.DismissBtn>
+        }
+        data-h2-radius="b(round)"
+        data-h2-padding="b(all, .25)"
+      >
+        <Alert.Title>
+          {text("title", "Oops! Something went wrong.")}
+        </Alert.Title>
+        <p>
+          {text(
+            "content",
+            "Please note the information here before continuing.",
+          )}
+        </p>
+      </Alert>
+    </section>
```

Comment:
Fair suggestion. I did it like this because I thought it seemed simpler for a user to implement. Basically, the dismiss button needs to exist outside of a content div that wraps both the title and other children. To pass the dismiss button in as children, we'd need to make a Alert.Content sub-component available as well, and using it would look like this:
```
<Alert>
   <Alert.DismissBtn onClick={action("Dismiss")}>x</Alert.DismissBtn>
   <Alert.Content>
       <Alert.Title>Title</Alert.Title>
      <p>Other content</p>
   </Alert.Content>
</Alert>
```
Its not bad, but its one more thing the user needs to know how to position.

Reply:
Oh I see what you mean. Ya I guess it's fine the way it is then 👍